crates/astria-composer/tests/blackbox/composer.rs

```diff
@@ -54,6 +60,46 @@ async fn tx_from_one_rollup_is_received_by_sequencer() {
     .expect("mocked sequencer should have received a broadcast message from composer");
 }
 
+#[tokio::test]
+async fn tx_from_one_rollup_is_received_by_sequencer_from_grpc_collector() {
+    let test_composer = spawn_composer(&[]).await;
+    tokio::time::timeout(
+        Duration::from_millis(100),
+        test_composer.setup_guard.wait_until_satisfied(),
+    )
+    .await
+    .expect("setup guard failed");
+
+    let rollup_id = RollupId::from_unhashed_bytes("test1");
+    let expected_chain_ids = vec![rollup_id];
+    let mock_guard =
+        mount_broadcast_tx_sync_mock(&test_composer.sequencer, expected_chain_ids, vec![0]).await;
+
+    let tx = Transaction::default();
+    // send sequence action request to the grpc generic collector
+    let mut composer_client = GrpcCollectorServiceClient::connect(format!(
+        "http://{}",
+        test_composer.grpc_collector_addr
+    ))
+    .await
+    .unwrap();
+    composer_client
+        .submit_rollup_transaction(SubmitRollupTransactionRequest {
+            rollup_id: rollup_id.as_ref().to_vec(),
+            data: tx.rlp().to_vec(),
+        })
+        .await
+        .expect("error submitting sequence actions to generic collector");
```

Comment:
The `expect` message should tell you that something failed, but instead what you expected to happen:
```suggestion
        .expect("can submit a transaction to composer's GrpcCollector service");
```
have a read here: https://doc.rust-lang.org/std/error/index.html#common-message-styles